test/test.js

```diff
@@ -57,3 +58,19 @@ assert.equal(commentedJSON.envOption, 42)
 assert.equal(commentedJSON.config, jsonrc)
 assert.equal(commentedJSON.configs.length, 1)
 assert.equal(commentedJSON.configs[0], jsonrc)
+
+// check if passing something other than an object as second param throws error
+errorWorthyTypes = ['asdf', 1, [], function() { }]
+
+for (var i = 0; i < errorWorthyTypes.length; i++) {
+    var errorThrown = false;
+    try {
+        rc(n, errorWorthyTypes[i]);
+    }
+    catch (err) {
+        errorThrown = true;
+    }
+
+    assert(errorThrown, true)
```

Comment:
Shouldn't that be `assert.equal(errorThrown, true)` or just `assert(errorThrown)`?